core/spec/models/spree/preferences/preferable_spec.rb

```diff
@@ -105,6 +105,46 @@ class B < A
       })
     end
 
+    describe '#admin_form_preference_field_types' do
+      subject { C.new.admin_form_preference_field_types }
+
+      before do
+        class C
```

Comment:
`C` is very short, and maybe someone else creates an example class `C`, with different behaviour. In order to avoid conflicts and possibly test order issues because of the conflict, you might want to consider naming the class `ClassWithPreferences` or the like.

Reply:
Uh-oh, there's also `A`, `B`, and `D`. Not concern of this PR, but 🗡